tests/fuzz/runFuzzTest.sh

```diff
@@ -59,12 +59,7 @@ fi
 FIPS_TEST_MSG=""
 if [ -n "${S2N_TEST_IN_FIPS_MODE}" ];
 then
-    if [[ $TEST_NAME == *bike* ]] || [[ $TEST_NAME == *sike* ]] || [[ $TEST_NAME == *kyber* ]]; then
-        printf "Skipping %s because PQ crypto is not supported in FIPS mode...\n" ${TEST_NAME}
-        exit 0
-    else
-        FIPS_TEST_MSG=" FIPS test"
-    fi
+  FIPS_TEST_MSG=" FIPS test"
```

Comment:
Nit: it looks like you copied the 2-space indent from line 67, but the rest of the script uses 4-space indents :\